blog/2023-12-22-openhab-4-1-release.md

```diff
@@ -0,0 +1,295 @@
+
+Six years after the introduction of Thing discovery, we are proud to announce add-on suggestions!
+
+When using the Setup Wizard or the add-on store, openHAB now suggests add-ons based on the devices found in your network and the programs installed on the openHAB server.
+This enhances the setup experience, making it more user-friendly, especially for those just getting started with openHAB.
+
+The add-on suggestions feature currently checks mDNS and UPnP traffic and performs a simple IP scan on your network to find out which devices you have at home.
+Additionally, the list of running processes of your openHAB server is checked to suggest for example persistence services like InfluxDB.
+
+Many thanks to Mark Herwege ([@mherwege](https://github.com/mherwege)), Andrew Fiddian-Green ([@andrewfg](https://github.com/andrewfg)) and Holger Friedrich ([@holgerfriedrich](https://github.com/holgerfriedrich)) for implementing that awesome new feature!
+
+## Core Runtime Enhancements
+
+### Extending the Ontology with your own Semantic Tags
+
+_Laurent Garnier, openHAB Maintainer_
+
+You can now easily extend the default ontology with your own semantic tags and then use them to build your semantic model.
+This can be used for example to add a location or equipment that does not (yet) exist by default.
+
+To define your own semantic tags, create a YAML configuration file containing the description of all your specific semantic tags in the `$OPENHAB_CONF/tags` folder.
+You can find a [tutorial on the community forum](https://community.openhab.org/t/oh-4-1-tutorial-to-manage-custom-semantic-tags/148135).
+
+### Voice System Enhancements
+
+_Miguel Álvarez Díez, openHAB Contributor_
+
+Some minor improvements were introduced in the built-in voice system components:
+
+The standard text interpreter now uses the Item's synonyms metadata as alternative names, which helps to avoid name collisions.
+It also automatically scans your Item's command description metadata, so that the rule `Set <ItemLabel> to <ItemCommandLabel>` is supported out-of-the-box for all supported languages.
+
+Additionally, the `voiceSystem` metadata namespace was created to allow adding custom rules, for example:
+
+- You can add the rule `go to mode $cmd$ on $name$` to an Item labeled `Vacuum` with command description `MODE_CLEAN=cleaning`:
+  That will instruct the interpreter to send the command `MODE_CLEAN` to the Item on the phrase `go to mode cleaning on vacuum`.
+- You can also capture free text with a rule like `watch $*$ on $name$`, so that the phrase `watch my favorite show on tv` will send the command `my favorite show` to the Item labeled as `TV`.
+
+Two new configuration parameters where added to the dialog processor start/register command:
+
+- The `dialogGroup` option prevents simultaneous processing of dialogs on the same group, so that several speakers located near-by won't be triggered at the same time.
```

Comment:
```suggestion
- The `dialogGroup` option prevents simultaneous processing of dialogs on the same group, so that several speakers located nearby won't be triggered at the same time.
```